client/src/reader/components/Cover/Cover.js

```diff
@@ -55,9 +56,35 @@ const Cover = ({
   return (
     <BrowsingLayout>
       {adventure ? (
-        <div>
+        <div className={styles.coverContainer}>
           <h1>{title}</h1>
-          <div dangerouslySetInnerHTML={{ __html: intro }} />
+          <p>
+            Created by{' '}
+            <strong>
+              {authors.length > 1
+                ? authors.map(a => a.username).join(', ')
+                : authors[0].username}
+            </strong>
+          </p>
+          <div>
+            <img
+              className={styles.coverImage}
+              alt="story cover"
+              src={coverImage}
+            />
+          </div>
+          <div className={styles.genre}>
+            <p>
+              This adventure falls into the <strong>{genre.name}</strong> genre.
+            </p>
+          </div>
+          <div className={styles.descriptionContainer}>
+            <p>Description:</p>
+            <p
+              className={styles.description}
+              dangerouslySetInnerHTML={{ __html: intro }}
```

Comment:
Let's control this intro and just use  a string and forego author-directed formatting